rfcs/text/0041/user.yml

```diff
@@ -0,0 +1,170 @@
+---
+- name: user.profile.id
+  level: extended
+  type: keyword
+  example: 1234
+  description: User ID from the identity datasource.
+- name: user.profile.type
+  level: extended
+  type: keyword
+  example: Employee
+  description: Type of user account.
+- name: user.profile.status
+  level: extended
+  type: keyword
+  example: On board
+  description: Status of the user account.
+- name: user.profile.first_name
+  level: extended
+  type: keyword
+  example: First
+  description: First Name of the User.
+- name: user.profile.last_name
+  level: extended
+  type: keyword
+  example: Last
+  description: Last Name of the user.
+- name: user.profile.other_identities
+  level: extended
+  type: keyword
+  multi_fields:
+    - type: text
+  example: first.last@elk.elastic.co
+  description: Array of additional user identities (usually email addresses).
+- name: user.profile.manager
+  level: extended
+  type: keyword
+  example: John Doe
+  description: Assigned Manager for the user account.
+- name: user.profile.employee_type
+  level: extended
+  type: keyword
+  example: Regular
+  description: Further classification type for the user account.
```

Comment:
how it correlates to just `user.profile.type`
Should we add more details into description to resolve this question for ECS users?